package.json

```
@@ -27,7 +29,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "pretty-quick --staged",
+      "pre-push": "yarn test"
```

Mungkin `pre-push` nya `yarn export` kali ya?

```suggestion
      "pre-push": "yarn export"
```

---

Hooh

---

Nice, commit the suggestion then!